<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Covercard Eingabe</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Eingabe Covercard Nr</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCovercardEingabe"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonXmlLesen">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Patieten Info Lesen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlz">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Plz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOrt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGeb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nachname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNachname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Strasse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAHV">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vorname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Geb.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStrasse">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AHV Nr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVorname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCovercardNr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Covercard</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSpeichern">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
